pkgs/development/tools/mold/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, lib, autoPatchelfHook, clang_12, cmake, lld_12, tbb, xxHash, zlib, openssl, git }:
+stdenv.mkDerivation rec {
+  pname = "mold";
+  version = "0.9.1";
+  meta = {
```

Comment:
Could you move meta section to end of expression? This will increase readability.